[0m15:46:39.318897 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F2BA2D9340>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F2BCAD3D40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F2BCB07E90>]}


============================== 15:46:39.325892 | ffc9a755-5950-4a0c-8a40-62e70c14d3fd ==============================
[0m15:46:39.325892 [info ] [MainThread]: Running with dbt=1.8.7
[0m15:46:39.328631 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'debug': 'False', 'profiles_dir': 'C:\\Users\\habteyes.asfaw\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'invocation_command': 'dbt init ethiopian_medical_businesses', 'static_parser': 'True', 'introspect': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m15:46:39.347942 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse dict {'dir': WindowsPath('C:/Users/habteyes.asfaw/.dbt')}
[0m15:46:39.348946 [info ] [MainThread]: Creating dbt configuration folder at 
[0m15:46:39.350086 [error] [MainThread]: Encountered an error:
'NoneType' object is not subscriptable
[0m15:46:39.350086 [error] [MainThread]: Traceback (most recent call last):
  File "C:\Users\habteyes.asfaw\10Accadamy\ethiopian-medical-data-warehouse\venv\Lib\site-packages\dbt\cli\requires.py", line 138, in wrapper
    result, success = func(*args, **kwargs)
                      ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\habteyes.asfaw\10Accadamy\ethiopian-medical-data-warehouse\venv\Lib\site-packages\dbt\cli\requires.py", line 101, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\habteyes.asfaw\10Accadamy\ethiopian-medical-data-warehouse\venv\Lib\site-packages\dbt\cli\main.py", line 470, in init
    results = task.run()
              ^^^^^^^^^^
  File "C:\Users\habteyes.asfaw\10Accadamy\ethiopian-medical-data-warehouse\venv\Lib\site-packages\dbt\task\init.py", line 320, in run
    profile_name = self.get_profile_name_from_current_project()
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\habteyes.asfaw\10Accadamy\ethiopian-medical-data-warehouse\venv\Lib\site-packages\dbt\task\init.py", line 141, in get_profile_name_from_current_project
    return dbt_project["profile"]
           ~~~~~~~~~~~^^^^^^^^^^^
TypeError: 'NoneType' object is not subscriptable

[0m15:46:39.359206 [debug] [MainThread]: Command `dbt init` failed at 15:46:39.359206 after 0.28 seconds
[0m15:46:39.359206 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F2BD5261B0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F2BE56F140>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F2BE4FFFE0>]}
[0m15:46:39.360291 [debug] [MainThread]: Flushing usage events
[0m15:50:39.360191 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001EA205E7140>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001EA1ECC7050>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001EA1FA58E30>]}


============================== 15:50:39.375949 | b9605839-cdee-4f75-b16c-a10ae863ca4d ==============================
[0m15:50:39.375949 [info ] [MainThread]: Running with dbt=1.8.7
[0m15:50:39.375949 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\habteyes.asfaw\\.dbt', 'debug': 'False', 'warn_error': 'None', 'log_path': 'C:\\Users\\habteyes.asfaw\\10Accadamy\\ethiopian-medical-data-warehouse\\dbt_project\\logs', 'version_check': 'True', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'static_parser': 'True', 'invocation_command': 'dbt init ethiopian_medical_businesses', 'introspect': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m15:50:39.375949 [info ] [MainThread]: Setting up your profile.
[0m15:50:39.391493 [error] [MainThread]: Encountered an error:
Runtime Error
  No adapters available. Learn how to install an adapter by going to https://docs.getdbt.com/docs/connect-adapters#install-using-the-cli
[0m15:50:39.391493 [debug] [MainThread]: Command `dbt init` failed at 15:50:39.391493 after 0.26 seconds
[0m15:50:39.391493 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001EA20783F80>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001EA202A74A0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001EA20724080>]}
[0m15:50:39.391493 [debug] [MainThread]: Flushing usage events
[0m15:52:14.755932 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000254A386BE30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000254A2F91FD0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000254A2F93D70>]}


============================== 15:52:14.755932 | dbd5d239-464f-47a7-9708-7ad977bda405 ==============================
[0m15:52:14.755932 [info ] [MainThread]: Running with dbt=1.8.7
[0m15:52:14.755932 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\habteyes.asfaw\\.dbt', 'fail_fast': 'False', 'warn_error': 'None', 'log_path': 'C:\\Users\\habteyes.asfaw\\10Accadamy\\ethiopian-medical-data-warehouse\\dbt_project\\logs', 'version_check': 'True', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt init ethiopian_medical_businesses', 'log_format': 'default', 'static_parser': 'True', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m15:52:14.776643 [info ] [MainThread]: Setting up your profile.
[0m16:00:43.974712 [error] [MainThread]: Encountered an error:

[0m16:00:43.989865 [error] [MainThread]: Traceback (most recent call last):
  File "C:\Users\habteyes.asfaw\10Accadamy\ethiopian-medical-data-warehouse\venv\Lib\site-packages\dbt\cli\requires.py", line 138, in wrapper
    result, success = func(*args, **kwargs)
                      ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\habteyes.asfaw\10Accadamy\ethiopian-medical-data-warehouse\venv\Lib\site-packages\dbt\cli\requires.py", line 101, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\habteyes.asfaw\10Accadamy\ethiopian-medical-data-warehouse\venv\Lib\site-packages\dbt\cli\main.py", line 470, in init
    results = task.run()
              ^^^^^^^^^^
  File "C:\Users\habteyes.asfaw\10Accadamy\ethiopian-medical-data-warehouse\venv\Lib\site-packages\dbt\task\init.py", line 321, in run
    self.setup_profile(profile_name)
  File "C:\Users\habteyes.asfaw\10Accadamy\ethiopian-medical-data-warehouse\venv\Lib\site-packages\dbt\task\init.py", line 263, in setup_profile
    self.create_profile_from_target(adapter, profile_name=profile_name)
  File "C:\Users\habteyes.asfaw\10Accadamy\ethiopian-medical-data-warehouse\venv\Lib\site-packages\dbt\task\init.py", line 180, in create_profile_from_target
    self.create_profile_from_profile_template(profile_template, profile_name)
  File "C:\Users\habteyes.asfaw\10Accadamy\ethiopian-medical-data-warehouse\venv\Lib\site-packages\dbt\task\init.py", line 164, in create_profile_from_profile_template
    target = self.generate_target_from_input(prompts, initial_target)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\habteyes.asfaw\10Accadamy\ethiopian-medical-data-warehouse\venv\Lib\site-packages\dbt\task\init.py", line 130, in generate_target_from_input
    target[key] = click.prompt(
                  ^^^^^^^^^^^^^
  File "C:\Users\habteyes.asfaw\10Accadamy\ethiopian-medical-data-warehouse\venv\Lib\site-packages\click\termui.py", line 164, in prompt
    value = prompt_func(prompt)
            ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\habteyes.asfaw\10Accadamy\ethiopian-medical-data-warehouse\venv\Lib\site-packages\click\termui.py", line 147, in prompt_func
    raise Abort() from None
click.exceptions.Abort

[0m16:00:43.996908 [debug] [MainThread]: Command `dbt init` failed at 16:00:43.996908 after 509.42 seconds
[0m16:00:43.997898 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000254A447C620>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000254A4703020>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000254A4701F40>]}
[0m16:00:43.997898 [debug] [MainThread]: Flushing usage events
[0m16:00:57.163203 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F2A529FB00>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F2A529F7A0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F2A529E6F0>]}


============================== 16:00:57.163203 | eec79c1a-cca9-4362-ad9e-428d69e6ce53 ==============================
[0m16:00:57.163203 [info ] [MainThread]: Running with dbt=1.8.7
[0m16:00:57.163203 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\habteyes.asfaw\\.dbt', 'version_check': 'True', 'warn_error': 'None', 'log_path': 'C:\\Users\\habteyes.asfaw\\10Accadamy\\ethiopian-medical-data-warehouse\\dbt_project\\logs', 'fail_fast': 'False', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'invocation_command': 'dbt init ethiopian_medical_businesses', 'introspect': 'True', 'static_parser': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m16:00:57.200633 [info ] [MainThread]: Setting up your profile.
[0m16:08:59.758835 [info ] [MainThread]: Profile my_project_profile written to C:\Users\habteyes.asfaw\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m16:08:59.758835 [debug] [MainThread]: Command `dbt init` succeeded at 16:08:59.758835 after 482.81 seconds
[0m16:08:59.758835 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F2A485EE40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F2A5A27D40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001F2A493A360>]}
[0m16:08:59.758835 [debug] [MainThread]: Flushing usage events
